.gitlab-ci.yml

```diff
@@ -0,0 +1,55 @@
+include:
+  - project: 'city-of-helsinki/kuva/ci-cd-config/ci-configuration'
+    ref: v1
+    file: '/.gitlab-ci-template.yml'
+
+variables:
+  SERVICE_PORT: 80
+  DOCKER_BUILD_ARG_REACT_APP_OIDC_CLIENT_ID: "https://api.hel.fi/auth/kukkuu-ui"
+
+review:
+  # These variables are available only for review env and are merged with the general variables defined above.
+  extends: .build
+  variables:
+    DOCKER_IMAGE_NAME: "$CI_PROJECT_NAME-review"
+    DOCKER_BUILD_ARG_REACT_APP_OIDC_AUTHORITY: "https://tunnistamo.test.kuva.hel.ninja"
+    DOCKER_BUILD_ARG_REACT_APP_OIDC_REDIRECT_URI: "https://kukkuu-ui.test.kuva.hel.ninja/callback" # FIXME This is a reference to the UI environment itself, is there a gitlab ci variable for this? 
+    DOCKER_BUILD_ARG_REACT_APP_PROFILE_GRAPHQL: "https://kukkuu.test.hel.ninja/graphql"
```

Comment:
Rename `DOCKER_BUILD_ARG_REACT_APP_API_URI` -> `DOCKER_BUILD_ARG_REACT_APP_API_URI`

OR

rename `REACT_APP_API_URI` in Dockerfile and client.ts to `REACT_APP_PROFILE_GRAPHQL`

Reply:
Oh, 🤦🏻‍♂️thank you!